presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator resource group aggregation issue related to WAITING_FOR_PREREQUISITE queries.
+* Fix Iceberg $files table in case of column dropping.
+* Fix a bug where ``EXPLAIN (TYPE IO)`` did not include the index source from an index join.
+* Fix a memory overcounting issue for using varchar types that could cause queries to fail with exceeded memory limit errors.
+* Fix an error in ``tdigest_agg``.
```

Comment:
Functions have to be written with  `:func:`

Reply:
Can we elaborate more about the "error"?

Reply:
add some description for the error:
Fix an off-by-one error during buffer size calculation in :func:`tdigest_agg`.